tests/ci_build/Dockerfile.cpu

```diff
@@ -23,7 +23,8 @@ ENV GOSU_VERSION 1.10
 RUN \
     pip install pyyaml cpplint pylint astroid sphinx numpy scipy pandas matplotlib sh recommonmark guzzle_sphinx_theme mock \
                 breathe matplotlib graphviz pytest scikit-learn wheel kubernetes urllib3 && \
-    pip install https://h2o-release.s3.amazonaws.com/datatable/stable/datatable-0.7.0/datatable-0.7.0-cp37-cp37m-linux_x86_64.whl
+    pip install https://h2o-release.s3.amazonaws.com/datatable/stable/datatable-0.7.0/datatable-0.7.0-cp37-cp37m-linux_x86_64.whl && \
+    conda install dask distributed -c conda-forge
```

Comment:
You can drop `distributed`.  The `dask` package is a meta package with includes distributed.  If you want just the core dask library then you would `conda install dask-core`

Reply:
This confuses me a little. I copied this line directly from your install documentation. Is this documentation out of date?

Reply:
Where were you looking?

Official docs are here: https://docs.dask.org/en/latest/install.html

I wouldn't be surprised if other docs exist elsewhere though

Reply:
I think google led me to some different instructions :)

Reply:
http://distributed.dask.org/en/latest/install.html

Reply:
Indeed, I just tried that and got there as a first hit.  We should probably start retiring a lot of content on that site.

Reply:
I think this would help your usability a lot. When I first tried dask I had a few install frustrations just because of old and inconsistent documentation. The other significant usability issue that I came across was errors because I didn't start a client within a main function. This was not clear to me and I think most users experience this and have two Google for the answer.

Reply:
- https://github.com/dask/dask/issues/4813
-  https://github.com/dask/distributed/issues/2708